### PR TITLE
add changelogs from v1.6.0

### PR DESCRIPTION
**Pull request by ansis**
Copy over changelogs.
Also update style-spec to a new dev version.

### CODE REVIEW

File: src/style-spec/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 13.10.0
+
+### ✨ Features and improvements
```
Comment (chloekraw) [inline src/style-spec/CHANGELOG.md]: The style spec changelog doesn’t have the tweaks made to the 1.6.0 changelog. If there is an opportunity to edit it, I think we should re-copy those over

Comment (ansis) [reply src/style-spec/CHANGELOG.md]: done in 8424208

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+* Add support for stretchable images (aka nine-part or nine-patch images). Stretchable images can be used with `icon-text-fit` to draw resized images with unstretched corners and borders. ([#8997](https://github.com/mapbox/mapbox-gl-js/pull/8997))
+* Add `in` expression. It can check if a value is in an array (`["in", value, array]`) or a substring is in a string (`["in", substring, string]`) ([#8876](https://github.com/mapbox/mapbox-gl-js/pull/8876))
+* Add `minPitch` and `maxPitch` map options ([#8834](https://github.com/mapbox/mapbox-gl-js/pull/8834))
+* Add `rotation`, `rotationAlignment` and `pitchAlignment` options to markers ([#8836](https://github.com/mapbox/mapbox-gl-js/pull/8836)) (h/t [dburnsii](https://github.com/dburnsii))
+* Add methods to Popup to manipulate container class names ([#8759](https://github.com/mapbox/mapbox-gl-js/pull/8759)) (h/t [Ashot-KR](https://github.com/Ashot-KR))
+* Add configurable inertia settings for panning (h/t @aMoniker) ([#8887](https://github.com/mapbox/mapbox-gl-js/pull/8887))
+* Add ability to localize mapbox UI controls ([#8095](https://github.com/mapbox/mapbox-gl-js/pull/8095)) (h/t [dmytro-gokun](https://github.com/dmytro-gokun))
```
Comment (chloekraw) [inline CHANGELOG.md]: Should Mapbox be capitalized?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,47 @@
...
+* Add LatLngBounds.contains() method ([#7512](https://github.com/mapbox/mapbox-gl-js/issues/7512), fixed by [#8200](https://github.com/mapbox/mapbox-gl-js/pull/8200))
+* Add option to load rtl-text-plugin lazily ([#8865](https://github.com/mapbox/mapbox-gl-js/pull/8865))
+* Add `essential` parameter to AnimationOptions that can override `prefers-reduced-motion: reduce` ([#8743](https://github.com/mapbox/mapbox-gl-js/issues/8743), fixed by [#8883](https://github.com/mapbox/mapbox-gl-js/pull/8883))
+
+## 🍏 Improvements
+* Allow rendering full world smaller than 512px. To restore the previous limit call `map.setMinZoom(0)` ([#9028](https://github.com/mapbox/mapbox-gl-js/pull/9028))
+* Add an es modules build of for mapbox-gl-style-spec in dist/ ([#8247](https://github.com/mapbox/mapbox-gl-js/pull/8247)) (h/t [ahocevar](https://github.com/ahocevar))
```
Comment (chloekraw) [inline CHANGELOG.md]: Typo: build of for

File: CHANGELOG.md (hunk 2 above)
Comment (ansis) [reply CHANGELOG.md]: I removed "mapbox" completely

File: CHANGELOG.md (hunk 3 above)
Comment (ansis) [reply CHANGELOG.md]: fixed

**Pull request merged**